src/views/dialogs/MobileDownloadDialog.tsx

```diff
@@ -0,0 +1,136 @@
+import styled, { AnyStyledComponent, css } from 'styled-components';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Dialog } from '@/components/Dialog';
+import { QrCode } from '@/components/QrCode';
+import { useStringGetter } from '@/hooks';
+import { STRING_KEYS } from '@/constants/localization';
+
+type ElementProps = {
+  setIsOpen: (open: boolean) => void;
+};
+
+/*
+  When/if deployer deploys the web app with smartbanner, "smartbanner:button-url-apple" and/or 
+  "smartbanner:button-url-google" <meta> are set.
+  This implementation assumes "smartbanner:button-url-apple" and "smartbanner:button-url-google" 
+  are set to the same value with onelink or other redirect URL.
+  Since there is no way for the desktop web app to know what mobile device the user is using, 
+  we should give a onelink URL which redirects to either iOS or Android app store depending on 
+  the mobile device used to scan the link.
+*/
+
+// for testing only
+// export const mobileAppUrl = "http://example.com";
+
+// for testing to verify <meta> is retrieved by name, QR code should show "@dYdX" as value
+// export const mobileAppUrl = document.querySelector('meta[name="twitter:creator"]')?.getAttribute('content')
+
+export const mobileAppUrl =
```

Comment:
Let's make this a function `getMobileAppUrl`, and call it when we need it.

Right now the query is being called when the the UI bundle isloaded rather than when we need in the component. Not sure if that would lead to race conditions